src/components/NewDesign/Modals/SelectModals/SelectTokenModal/SelectTokenModal.tsx

```diff
@@ -4,10 +4,9 @@ import CustomScrollbar from '../CustomScrollbar'
 import icons from '@static/icons'
 import useStyles from '../style'
 import searchIcon from '@static/svg/lupa.svg'
-import { BN } from '@project-serum/anchor'
 export interface ISelectTokenModal {
-  tokens: Array<{ symbol: string; balance?: BN; decimals?: number }>
-  commonTokens: Array<{ symbol: string; balance?: BN; decimals?: number }>
+  tokens: Array<{ symbol: string } | null>
```

Comment:
I have an idea. Because we will later fetch tokens' symbols, names and icons from Solana's tokens list, use this prop to pass these instead of for example hardcoding object with names of tokens

Reply:
done